packages/docusaurus-protobuffet-plugin/src/theme/ProtoFile.tsx

```diff
@@ -144,6 +144,9 @@ const ProtoMessageFields = (props: MessageFieldsProps) => {
     );
 
     return (
+      { if(field.label != "") {
```

Comment:
there are some code formatting issues here that will break things. can we conditionally inject this in a different way?